cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```
@@ -587,5 +587,14 @@ data:
         }
     }
 
+    location /model/installInfo {
+            default_type 'application/json';
+            return 200 '\n
+                {
+                    "version": "{{ .Chart.Version }}"
+                }
+            ';
+        }
```

Looks good! Minor indentation/style change? I've found that when this Nginx conf is set to return JSON, it does a poor job with newlines and indentation. I say we let the client-side prettify the JSON.

```suggestion
    location /model/installInfo {
        default_type 'application/json';
        return 200 '{"version": "{{ .Chart.Version }}"}';
    }
```